.github/workflows/build-docker-image.yml

```diff
@@ -0,0 +1,47 @@
+name: Build and Publish Docker image
+
+on:
+  push:
+    branches:
+      - develop
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+      - "v[0-9]+.[0-9]+.[0-9]+-pre[0-9]+"
```

Comment:
why only these patterns?

Reply:
Can't we build image for any pattern which are correct for semver basically?

Reply:
Should I change to any tag?

Reply:
I would think so, it's not unusal to have any kind of tag, look at our staging environment ust now
<img width="426" alt="Screenshot 2021-12-29 at 22 37 49" src="https://user-images.githubusercontent.com/4346460/147708484-a8eb1ffd-fbea-4eb4-8844-f0d9f3ef3c30.png">